executor/prepared.go

```diff
@@ -145,7 +145,7 @@ func (e *PrepareExec) Next(ctx context.Context, req *chunk.Chunk) error {
 	e.ParamCount = paramCnt
 	e.Stmt = stmt
 	if e.IsGeneralStmt {
```

Comment:
should  the `IsGeneralStmt`  also be renamed？

Reply:
Updated, PTAL